patches/chromium/fix_add_support_for_webauthn_on_macos.patch

```diff
@@ -0,0 +1,315 @@
+index 60c4c6abefd4ffcbb30374dd8d455c5dea3a75ae..2bb3aec2abc0488c814d443681806ca6d7ed17a1 100644
+--- a/chrome/browser/webauthn/authenticator_request_dialog_model.cc
++++ b/chrome/browser/webauthn/authenticator_request_dialog_model.cc
+@@ -15,12 +15,12 @@
+ #include "base/threading/sequenced_task_runner_handle.h"
+ #include "build/build_config.h"
+ #include "chrome/app/vector_icons/vector_icons.h"
+-#include "chrome/grit/generated_resources.h"
+ #include "components/vector_icons/vector_icons.h"
+ #include "device/fido/features.h"
+ #include "device/fido/fido_authenticator.h"
+ #include "device/fido/pin.h"
+ #include "device/fido/public_key_credential_user_entity.h"
++#include "electron/grit/electron_resources.h"
+ #include "ui/base/l10n/l10n_util.h"
+ #include "ui/gfx/text_elider.h"
+ 
+diff --git a/chrome/browser/webauthn/chrome_authenticator_request_delegate.cc b/chrome/browser/webauthn/chrome_authenticator_request_delegate.cc
+index 2becd355dbd04d80ce9effe0ae21778e7d181a42..4c1a10abb684dfc855ed554ae49d45ed6a4098aa 100644
+--- a/chrome/browser/webauthn/chrome_authenticator_request_delegate.cc
++++ b/chrome/browser/webauthn/chrome_authenticator_request_delegate.cc
+@@ -21,7 +21,6 @@
+ #include "build/chromeos_buildflags.h"
+ #include "chrome/browser/extensions/api/web_authentication_proxy/web_authentication_proxy_service.h"
+ #include "chrome/browser/net/system_network_context_manager.h"
+-#include "chrome/browser/profiles/profile.h"
+ #include "chrome/browser/sync/device_info_sync_service_factory.h"
+ #include "chrome/browser/ui/browser.h"
+ #include "chrome/browser/ui/browser_finder.h"
+@@ -32,7 +31,6 @@
+ #include "chrome/common/chrome_switches.h"
+ #include "chrome/common/chrome_version.h"
+ #include "chrome/common/pref_names.h"
+-#include "chrome/grit/generated_resources.h"
+ #include "components/device_event_log/device_event_log.h"
+ #include "components/pref_registry/pref_registry_syncable.h"
+ #include "components/prefs/pref_service.h"
+@@ -50,6 +48,8 @@
+ #include "device/fido/features.h"
+ #include "device/fido/fido_authenticator.h"
+ #include "device/fido/fido_discovery_factory.h"
++#include "electron/grit/electron_resources.h"
++#include "electron/shell/browser/electron_browser_context.h"
+ #include "ui/base/l10n/l10n_util.h"
+ 
+ #if defined(OS_MAC)
+@@ -66,6 +66,8 @@
+ #include "ui/aura/window.h"
+ #endif
+ 
++using electron::ElectronBrowserContext;
++
+ namespace {
+ 
+ ChromeAuthenticatorRequestDelegate::TestObserver* g_observer = nullptr;
+@@ -75,8 +77,8 @@ ChromeAuthenticatorRequestDelegate::TestObserver* g_observer = nullptr;
+ bool IsWebauthnRPIDListedInEnterprisePolicy(
+     content::BrowserContext* browser_context,
+     const std::string& relying_party_id) {
+-  const Profile* profile = Profile::FromBrowserContext(browser_context);
+-  const PrefService* prefs = profile->GetPrefs();
++  const auto* profile = static_cast<ElectronBrowserContext*>(browser_context);
++  const PrefService* prefs = profile->prefs();
```

Comment:
🤔  minor: perhaps we should add a `GetPrefs()` method to `ElectronBrowserContext` to slightly reduce the patch surface area here?

Reply:
Seems reasonable

Reply:
I want ahead and did this in a03f822, compiles fine and reduces the chromium patch a bit.